packages/web/src/components/Plan.tsx

```diff
@@ -337,7 +364,7 @@ export class Plan extends React.Component<{}, State> {
                                 >
                                     <Button
                                         type="primary"
-                                        onClick={this.addEvent}
+                                        onClick={this.addTutorial}
```

Comment:
This should add a new blank default event, not a tutorial one.

Tutorial event only shows on the *first* run when there's nothing saved in the user's browser.  Subsequent events added shouldn't be a tutorial event.

Reply:
oh my b I was just testing the function, I'll change it back